doc/source/ray-core/scheduling/configure-memory-monitor.rst

```diff
@@ -0,0 +1,39 @@
+Configure memory monitor
+========================
+
+The memory monitor periodically checks the memory usage of the node and reports whether 
+the usage is above the threshold. The raylet uses this signal to evict worker processes to reduce the memory usage. This allows Ray to pre-emptively shed
```

Comment:
Can you mention if it is on or off by default earlier? (and add a link to the configuration section)